content/docs/manifest.json

```
@@ -472,6 +476,10 @@
               "title": "Developing with Kind",
               "path": "/docs/contributing/kind.md"
             },
+            {
+              "title": "Implementing feature gates",
```

nitpick: The rest of the titles in this section use title case so this one stands out a little

```suggestion
              "title": "Implementing Feature Gates",
```